mirage/factories/brand-assets.ts

```diff
@@ -0,0 +1,23 @@
+import { Factory } from 'ember-cli-mirage';
+
+import BrandAssets from 'ember-osf-web/models/brand-assets';
+
+export default Factory.extend<BrandAssets>({
+    primaryColor: 'green',
+    secondaryColor: 'blue',
+    navbarLogoImage: 'http://somelogoimageurl',
+    heroLogoImage: 'http://somelogoimageurl',
+    heroBackgroundImage: 'http://herobackgroundimage',
+});
+
+declare module 'ember-cli-mirage/types/registries/model' {
+    export default interface MirageModelRegistry {
+        brandAssets: BrandAssets;
```

Comment:
```suggestion
        'brand-assets': BrandAssets;
```
for the type registries to work, the key needs to match the string used in string-based lookups